Compilation error for qtkeychain-0.9.1: QStringLiteral was not declared in this scope

gcc (GCC) 9.0.1 20190312 on Fedora Rawhide

Built with:
-DBUILD_WITH_QT4:BOOL=ON
-DCMAKE_BUILD_TYPE=RelWithDebInfo

```
BUILDSTDERR: /builddir/build/BUILD/qtkeychain-0.9.1/keychain_unix.cpp: In function 'bool isKwallet5Available()':
BUILDSTDERR: /builddir/build/BUILD/qtkeychain-0.9.1/keychain_unix.cpp:94:37: error: 'QStringLiteral' was not declared in this scope; did you mean 'QLatin1Literal'?
BUILDSTDERR:    94 |     QDBusMessage reply = iface.call(QStringLiteral("networkWallet"));
BUILDSTDERR:       |                                     ^~~~~~~~~~~~~~
BUILDSTDERR:       |                                     QLatin1Literal
BUILDSTDERR: make[2]: *** [CMakeFiles/qtkeychain.dir/build.make:148: CMakeFiles/qtkeychain.dir/keychain_unix.cpp.o] Error 1
BUILDSTDERR: make[2]: *** Waiting for unfinished jobs....
```

--- a/keychain_unix.cpp
+++ b/keychain_unix.cpp
@@ -93,7 +93,7 @@ static bool isKwallet5Available()
     // a wallet can be opened.
 
     iface.setTimeout(500);
-    QDBusMessage reply = iface.call(QStringLiteral("networkWallet"));
+    QDBusMessage reply = iface.call(QLatin1String("networkWallet"));
     return reply.type() == QDBusMessage::ReplyMessage;
 }
 

--- a/libsecret.cpp
+++ b/libsecret.cpp
@@ -305,7 +305,7 @@ bool LibSecretKeyring::deletePassword(const QString &key, const QString &service
 }
 
 LibSecretKeyring::LibSecretKeyring()
-    : QLibrary(QStringLiteral("secret-1"), 0)
+    : QLibrary(QLatin1String("secret-1"), 0)
 {
 #ifdef HAVE_LIBSECRET
     if (load()) {

